app/services/food_service.rb

```diff
@@ -0,0 +1,12 @@
+class FoodService
+  def self.conn 
+    Faraday.new("https://api.nal.usda.gov/fdc/v1/foods") do |faraday|
+      faraday.params['api_key'] = ENV['food_api_key']
+    end
+  end
+
+  def self.get_foods(ingredient)
+    response = conn.get('list', {query: ingredient})
```

Comment:
We were actually expecting you to hit the `/search` endpoint for this user story. While list is okay for the food items (more or less the same data) it doesn't provide the total number of hits we were looking for. We know it's hard to spend a lot of time reading the docs during an assessment though, so in the future when facing this question of which endpoint to use, when in a non-assessment format, it would be worth doing a bit more research to find the endpoint that returns all the data you need